AZ3166/src/cores/arduino/system/SystemVariables.cpp

```diff
@@ -24,6 +25,8 @@ int GetMACWithoutColon(char* buff)
         }
     }
 
+    LogInfo("DevKit MAC address: %s", buff);
```

Comment:
Do not call az_iot in system file
 Revert this  file

Reply:
I will use Serial.printf.